.gitignore

```diff
@@ -29,3 +29,4 @@ site/yarn-debug.log*
 site/yarn-error.log*
 site/static/.DS_Store
 site/temp
+/.idea/
```

Comment:
unnecessary change - already in here

Reply:
also you can have a global gitignore in your os... that's where people usually place the gitignore for something that's not generated by the project